card/mod/02_basic_types/set/all/base.rb

```diff
@@ -7,38 +7,37 @@ def show view, args
 
   # NAME VIEWS
 
-  view :name,     closed: true, perms: :none do |args| card.name                           end
-  view :key,      closed: true, perms: :none do |args| card.key                            end
-  view :title,    closed: true, perms: :none do |args| args[:title] || card.name           end
-
-  view :linkname, closed: true, perms: :none do |args| card.cardname.url_key               end
-  view :url,      closed: true, perms: :none do |args| card_url _render_linkname           end
-  view :url_link, closed: true, perms: :none do |args| web_link card_url(_render_linkname) end
+  simple_args = { closed: true, perms: :none }
+  view(:name,     simple_args) { card.name                           }
```

Comment:
I haven't gotten the local configs set up yet, but rubocop seems more permissive on this stuff locally.  (And it seems to matter whether it's done for alignment purposes)